packages/lambda-xyz/src/wmts-capability.ts

```diff
@@ -0,0 +1,162 @@
+import { V, VToString } from '@basemaps/lambda-shared';
+
+const CapabilitiesAttrs = {
```

Comment:
Why not creation this as a template and then populate the parameters from there?

Also note with clients like ArcGIS desktop and even QGIS, the order the bounding boxes (e.g WGS84 and tilematrix projection) is very important!

Reply:
By constructing the xml pragmatically it makes it easier to deploy the change and indent the xml correctly